core/shortcut_registry.js

```diff
@@ -352,4 +353,6 @@ Blockly.ShortcutRegistry.prototype.createSerializedKey = function(
 };
 
 // Creates and assigns the singleton instance.
-new Blockly.ShortcutRegistry();
+new ShortcutRegistry();
+
+exports = ShortcutRegistry;
```

Comment:
Fine for now (to keep API compatibility until the first pass is finished), but it should probably be changed to something like:
```JS
exports.shortcutRegistry = new ShortcutRegistry();
```
i.e., only exporting the singleton, not the constructor (and not having the constructor overwrite any previously-created singleton).

Added to the list in #5073.

Reply:
Yeah that does seem much nicer, good call!